metrics/vocabulary/vocabulary_test.go

```diff
@@ -57,186 +47,98 @@ func testVocabulary(t *testing.T, operation func(a []*metrics.Vocabulary) *metri
 }
 
 func TestSampleVocabularyUnion(t *testing.T) {
-	schemaWords := []string{"heelo", "random", "funcName", "google"}
-	schemaCount := []int{1, 2, 3, 4}
-	propWords := []string{"Hello", "dog", "funcName", "cat"}
-	propCount := []int{4, 3, 2, 1}
-	opsWords := []string{"countGreetings", "print", "funcName"}
-	opsCount := []int{12, 11, 4}
-	paramWords := []string{"name", "id", "tag", "suggester"}
-	paramCount := []int{5, 1, 1, 15}
-
 	v1 := metrics.Vocabulary{
-		Schemas:    fillTestProtoStructue(schemaWords, schemaCount),
-		Properties: fillTestProtoStructue(propWords, propCount),
-		Operations: fillTestProtoStructue(opsWords, opsCount),
-		Parameters: fillTestProtoStructue(paramWords, paramCount),
+		Schemas:    fillTestProtoStructue([]string{"heelo", "random", "funcName", "google"}, []int{1, 2, 3, 4}),
+		Properties: fillTestProtoStructue([]string{"Hello", "dog", "funcName", "cat"}, []int{4, 3, 2, 1}),
+		Operations: fillTestProtoStructue([]string{"countGreetings", "print", "funcName"}, []int{12, 11, 4}),
+		Parameters: fillTestProtoStructue([]string{"name", "id", "tag", "suggester"}, []int{5, 1, 1, 15}),
 	}
 
-	schemaWords2 := []string{"Hello", "random", "status", "google"}
-	schemaCount2 := []int{5, 6, 1, 4}
-	propWords2 := []string{"cat", "dog", "thing"}
-	propCount2 := []int{4, 3, 2}
-	opsWords2 := []string{"countPrint", "print", "funcName"}
-	opsCount2 := []int{17, 12, 19}
-	paramWords2 := []string{"name", "id", "tag", "suggester"}
-	paramCount2 := []int{5, 1, 1, 15}
-
 	v2 := metrics.Vocabulary{
-		Schemas:    fillTestProtoStructue(schemaWords2, schemaCount2),
-		Properties: fillTestProtoStructue(propWords2, propCount2),
-		Operations: fillTestProtoStructue(opsWords2, opsCount2),
-		Parameters: fillTestProtoStructue(paramWords2, paramCount2),
+		Schemas:    fillTestProtoStructue([]string{"Hello", "random", "status", "google"}, []int{5, 6, 1, 4}),
+		Properties: fillTestProtoStructue([]string{"cat", "dog", "thing"}, []int{4, 3, 2}),
+		Operations: fillTestProtoStructue([]string{"countPrint", "print", "funcName"}, []int{17, 12, 19}),
+		Parameters: fillTestProtoStructue([]string{"name", "id", "tag", "suggester"}, []int{5, 1, 1, 15}),
 	}
 
 	vocab := make([]*metrics.Vocabulary, 0)
```

Comment:
I think this slice (and similar ones) should be named "vocabs" or "vocabularies" to clearly show that they are collections and are not individual vocabularies.